src/full_client.toit

```diff
@@ -668,6 +668,9 @@ class Session_:
     while true:
       candidate := next-packet-id_
       next-packet-id_ = (candidate + 1) & ((1 << PublishPacket.ID-BIT-SIZE) - 1)
```

Comment:
I'd move this to just before returning candidate, but maybe that is just me.

Reply:
Since we get the next candidate from the next-packet-id_, I would need to store that counter locally.
I rather change the field, so I can't forget.